.travis.yml

```
@@ -17,7 +17,9 @@ addons:
 env:
   global:
   - VERSION="12.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
-
+  {%- if include_wkhtmltopdf %}
+  - WKHTMLTOPDF_VERSION="0.12.5"
+  {%- endif %}
```

Actually this is how it should look like.

```suggestion
  - WKHTMLTOPDF_VERSION="0.12.5"
```

`{% - ... - %}` are jinja expressions only used by the repository template.

Adding them here unfortunately has Travis not even run because of a malformed `.travis.yml` file, so in fact we don't know yet if it's fixed or not, because CI didn't run 😢